src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
+    //Main Program
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+        Scanner sc = new Scanner(System.in);
+        String command = sc.nextLine();
+        Task[] list = new Task[100];
+        int num = 0;
+
+        while(!command.equals("bye")) {
+            String actionType = extractAction(command); //Find out what action to be taken
+            switch (actionType) {
+            case "list": printList(list, num); //Print the list of tasks
+                break;
+            case "done": doneTask(list, command); //Put a tick in the task
+                break;
+            case "todo": num = storeTodo(list, command, num); //Add task under the "td" category
+                break;
+            case "deadline": num = storeDeadline(list, command, num); //Add task under the "deadline" category
+                break;
+            case "event": num = storeEvent(list, command, num); //Add task under the "event" category
+                break;
+            default : System.out.println("Error"); //Show error because invalid input
+                break;
+            }
+            System.out.println("\n");
+            command = sc.nextLine(); //Get the next command
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+    }
```

Comment:
Perhaps you can break this into methods like printHello, handleInput, to make the code more readable?

Reply:
Good! Follows all basic coding standards laid out in the syllabus. In the case of actionType, you can consider using constants to reduce magic literals.